### PR TITLE
Alteration in README.md

### LINKED ISSUE #79: [Feat]: Addition in README.md
### Detailed Description

I think the feature [Inline Links] should be added because in the [README.md](https://github.com/WebXDAO/DEV-NFT/blob/main/README.md) file, there are URLs which can be made to look more static and easy-to-read if made into inline links.
Also, The [Contributing Guidelines](https://github.com/WebXDAO/DEV-NFT/blob/main/CONTRIBUTING.md) page can be improved as I have observed a few errors in regards to the terms used.

### Contributing

- [X] I have read the project's contribution guidelines.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Anything else?

I would be very glad to be assigned this issue!

### PR DESCRIPTION
**Pull request by AlphaVS-76**
#### Description of the change\*
- Addition of a few inline links.
- Documentation altered to be more client-friendly

CLOSES #79

#### Checklist\*

**Please make sure you have done the following:**

- [x] 😄 I've read [CONTRIBUTING.md][contributing] and my PR is up to standards.
- [x] 🎨 The changes in this PR meet the design plan.
- [ ] 🐛 I've tested my PR's source changes for any bugs/render issues.
- [ ] 🧪 I've worked through build failures and tests are passing.
- [x] 📄 I've updated the necessary documentation changes.

#### Notes

- I suggest #79 should not be closed yet as I will be making another PR regarding changes in Contributing Guideline page.

[contributing]: https://github.com/web3community/DEV-NFT/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Review by krshdev — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,56 +2,58 @@
...
 
 We are currently on mumbai testnet. 
 To use DevNFT you need to setup mumbai testnet network on metamask. 
 - Go to metamask settings > add network > 
 - NetworkName: 	Mumbai
 - chainId:	80001
 - Gas Token:	MATIC Token
-- Rpc url: https://matic-mumbai.chainstacklabs.com
-- blockscan: https://mumbai.polygonscan.com/
+- [RPC url](https://matic-mumbai.chainstacklabs.com)
```
Comment (KukretiShubham) [inline README.md:21]: These kept for people so that they can paste the links while setting their metamask.

File: README.md [hunk 2]
```diff
@@ -2,56 +2,58 @@
...
+- [Blockscan](https://mumbai.polygonscan.com/)
```
Comment (KukretiShubham) [inline README.md:22]: same for this

Comment (AlphaVS-76) [reply README.md:22]: Ahh I see.. Alright I'll close this PR and make an updated one.

**Comment by KukretiShubham** [comment]
You can make change on the same PR as well 😄

**Pull request closed without merging**